modules/ROOT/pages/debugging-outside-studio.adoc

```diff
@@ -0,0 +1,40 @@
+= Debugging Outside Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: debugging, debug
+
+
+To troubleshoot a Mule app in Anypoint Studio, you can use the xref:studio::visual-debugger-concept.adoc[Studio Visual Debugger]. This lets you set breakpoints to stop execution and check events.
+
+If you're creating your Mule app outside of Studio, you can either use MUnit to test your app, or you can activate remote debugging.
```

Comment:
If you "are"...,  the rest looks good to me!

Reply:
I think our style is to use contractions like you're. I'll double-check on this,